src/components/layouts/Sidebar.tsx

```diff
@@ -32,8 +34,40 @@ const getValueForMenuItem = getValueFromTheme('menuItem');
 const getValueForSidebar = getValueFromTheme('sidebar');
 const getValueForMenu = getValueFromTheme('menu');
 
+const PermissionTypes = {
+  AANBOD_BEWERKEN: 'AANBOD_BEWERKEN',
```

Comment:
Did you consider using english names for the keys?

Reply:
I would also prefer english names, but here we use the same ones comming from the udb3-backend also in dutch